docs/tutorials/tfx/stub_template.md

```diff
@@ -0,0 +1,108 @@
+
+## Testing the pipeline using Stub Executors
+
+### Introduction
+**You should complete [template.ipynb](https://github.com/tensorflow/tfx/blob/master/docs/tutorials/tfx/template.ipynb) tutorial up to *6* in order to proceed this tutorial.**
+
+This document will provide instructions to test a TensorFlow Extended (TFX) pipeline
+using `BaseStubExecuctor`, which generates fake data using the golden test data. This is intended for users to replace executors they don't want to test so that they could save time from running the actual executors. Stub executor is provided with TFX Python package under `tfx.experimental.pipeline_testing.base_stub_executor`.
+
+This tutorial serves as an extension to `template.ipynb` tutorial, thus you will also use [Taxi Trips dataset](
+https://data.cityofchicago.org/Transportation/Taxi-Trips/wrvz-psew)
+released by the City of Chicago. We strongly encourage you to try modifying the components prior to utilizing stub executors.
+
+### 1. Record the pipeline outputs in Google Cloud Storage
```

Comment:
Would you mind adding some more explanation about why we need to *record* outputs?